src/fsharp/absil/ilread.fs

```diff
@@ -1719,6 +1719,7 @@ and typeDefReader ctxtH: ILTypeDefStored =
                      methodImpls=mimpls,
                      events= events,
                      properties=props,
+                     isAttribute=false,
```

Comment:
It looks like `IsAttribute` could only be set to `true` in `IlxGen` and only used for writing an assembly. This means every read `ILTypeDef` will have `IsAttribute` return false. We should probably add a comment to `ILTypeDef.IsAttribute` describing that it's only used for writing an assembly.

Reply:
> This means every read ILTypeDef will have IsAttribute return false.

Using a name like "IsDefinitelyAttribute" or "IsKnownToBeAttribute" may be clarifying